tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -177,6 +181,154 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, AllocateLoadComponentArraysTest )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, AllocateLoadComponentArraysTest" );
+
+	AllocateLoadComponentArraysDoAllocate = true;
```

Comment:
I thought EnergyPlusFixture was only needed when feeding IDF objects as strings. When should EnergyPlusFixture be used when writing unit tests?